.env.example

```
@@ -7,7 +7,22 @@ owner = ''
 # Database
-dbServer = ''
+# For MongoDB Atlas (recommended):
+# MONGO_DATABASE_URL = 'mongodb+srv://username:password@cluster.mongodb.net/database-name'
+
+# For local development (without Docker):
+# MONGO_DATABASE_URL = 'mongodb://localhost:27017/database-name'
+
+# For Docker's local MongoDB:
+# MONGO_DATABASE_URL = 'mongodb://username:password@mongodb:27017/database-name?authSource=admin'
+
+MONGO_DATABASE_URL = ''
```

dotenv-linter (4.0.0)

[warning] [SpaceCharacter] The line has spaces around equal sign

(SpaceCharacter)